templates/base.html

```diff
@@ -54,15 +54,18 @@
   {% block extrahead %}{% endblock %}
 
   {% if config.google_analytics_id %}
+    <script src="https://github.com/MohamedBassem/spam-referrals-blocker/releases/download/v0.2/spam-referrals-blocker.min.js"></script>
```

Comment:
I don't really like pulling in JS from a third-party like this. Could we host the file locally, or fork the repo? Or convince me I'm being silly?